kw6/stream.py

```diff
@@ -72,18 +104,13 @@ def _seek_closest_stored_position(self, frame_index: types.FRAME_INDEX):
             )
```

Comment:
You can just do a simple "max" since you do `<=` before

Reply:
Good point :) Fixed